uk.co.compucorp.civicrm.tasksassignments/tests/phpunit/api/v3/DocumentTest.php

```diff
@@ -15,19 +17,15 @@ class api_v3_DocumentTest extends CiviUnitTestCase {
   /**
    * Runs installer and sets default document type
    */
-  public function setUp() {
-    parent::setUp();
-    $this->quickCleanup(['civicrm_activity', 'civicrm_case', 'civicrm_case_activity'], TRUE);
-    $upgrader = CRM_Tasksassignments_Upgrader::instance();
-    $upgrader->install();
-
-    $hrCaseUpgrader = CRM_HRCase_Upgrader::instance();
-    $hrCaseUpgrader->install();
+  public function setUpHeadless() {
```

Comment:
Do we still need this since the test class is extending BaseHeadlessTest?

Reply:
Was removed